tensorflow/core/kernels/depthwise_conv_op_gpu.h

```diff
@@ -323,7 +322,7 @@ __global__ __launch_bounds__(1024, 2) void DepthwiseConv2dGPUKernelNHWCSmall(
   }
 }
 
-// A Cuda kernel to compute the depthwise convolution forward pass
+// A Gpu kernel to compute the depthwise convolution forward pass
```

Comment:
GPU

Reply:
Ooops, rebased last commit, all Cuda/Gpu updated to GPU in comments